app/assets/stylesheets/pages/_landing-page.scss

```diff
@@ -172,11 +172,40 @@ body {
   height: 100%;
 }
 
-.landing_page {
-}
+#user-info {
```

Comment:
Move this to _userinfo_box.scss ?

Reply:
Updated to components/_user_info_box.scss